phpunit.xml.dist

```diff
@@ -48,4 +48,8 @@
             <directory>tests</directory>
         </testsuite>
     </testsuites>
+
```

Comment:
I never used this config, does it makes sense to keep it?

Reply:
This config is used by default.